app/src/main/java/org/simple/clinic/scanid/PatientPrefillInfoPayload.kt

```diff
@@ -0,0 +1,32 @@
+package org.simple.clinic.scanid
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class PatientPrefillInfoPayload(
```

Comment:
Let's rename this to `IndiaNHIDPayload` because this payload object is specific to India. That way in the future we can add more countries if needed. WDYT?